api.json

```diff
@@ -1359,6 +1359,57 @@
         }
       }
     },
+    "/balances/gift_cards": {
+      "get": {
+        "description": "Returns a paginated list of gift cards.",
+        "summary": "List gift cards",
+        "tags": [
+          "Gift Cards"
+        ],
+        "operationId": "ListGiftCards",
+        "parameters": [
+          {
+            "in": "query",
+            "name": "before",
+            "type": "string",
+            "description": "The upper limit for the ID to be included in the response."
+          },
+          {
+            "in": "query",
+            "name": "page_size",
+            "type": "integer",
+            "description": "The maximum number of items to be returned in the response."
```

Comment:
add "NOTE: 200 is the maximum value of the page_size parameter."
(this is in the pagination section of the reference anyway but won't hurt to point it out here as well)

Reply:
done, ta